inference-engine/tests/functional/plugin/gna/pass_tests/layers_restrictions.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
As soon as I know pass_tests are related to verification of GNA passes but this test isn't related to passes verification. Maybe it is reasonable to create a separate directory for it (for example layers_restrictions and the file name will be something like affine_batch_size_restrictions).

Reply:
AreLayersSupported() is used only in gna_plugin.cpp after GNA passes. So these tests are related with passes. I suggest not moving tests.

Reply:
From my point of view verification of GNA restrictions and verification of passes are different things. So it's not evident that restrictions tests can be found in passes tests, that's quite confusing.